src/Speller.php

```diff
@@ -107,6 +111,34 @@ public static function spellNumber($number, $language)
 			->parseInt(intval($number), false);
 	}
 	
+	/**
+	 * Convert currency to its linguistic representation.
+	 *
+	 * @param int|float $amount : the amount to spell in the specified language
+	 * @param string $language : a two-letter, ISO 639-1 code of the language to spell the amount in
+	 * @param string $currency : a three-letter, ISO 4217 currency code
+	 * @return string : the currency as written in words in the specified language
+	 * @throws InvalidArgumentException if any parameter is invalid
+	 */
+	public static function spellShort($amount, $language, $currency)
+	{
+		$amount = is_numeric($amount) ? $amount : 0;
```

Comment:
Use the same checks and exceptions as in `spellCurrency`.

Reply:
corrected